packages/language-service/src/refactorings/convert_to_signal_input.ts

```
@@ -183,6 +193,18 @@ export class ConvertToSignalInputRefactoring implements ActiveRefactoring {
   }
 }
 
+export class ConvertToSignalInputRefactoring extends BaseConvertToSignalInputRefactoring {
+  static id = 'convert-to-signal-input-safe-mode';
+  static description = 'Convert this `@Input()` to a signal input';
```

```suggestion
  static description = 'Convert this `@Input()` to a signal input (safe)';
```

---

Maybe differentiate the two with just a "(safe)" and "(aggressive)" suffix?

---

I like that. And then when failing safe we could recommend aggresive.

---

Done. Would also like to have @MarkTechson's feedback here.

---

I think the descriptions are good based on the screenshot. I would consider (forcibly, ignoring errors)